src/popup/router/pages/Names/Auction.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="auction">
+    <Plate>
+      <AccountInfo />
+    </Plate>
+    <Tabs>
+      <RouterLink
+        :to="{ name: 'auction-bid' }"
+        exact-path
+      >
+        <Auction /> {{ $t('pages.names.auctions.place-bid') }}
+      </RouterLink>
+      <RouterLink :to="{ name: 'auction-history' }">
+        <History /> {{ $t('pages.names.auctions.bid-history') }}
+      </RouterLink>
+    </Tabs>
+    <Loader v-if="loading" />
+    <RouterView v-else />
+  </div>
+</template>
+
+<script>
+import AccountInfo from '../../components/AccountInfo';
+import Plate from '../../components/Plate';
+import Tabs from '../../components/Tabs';
+import Auction from '../../../../icons/auction.svg?vue-component';
+import History from '../../../../icons/history.svg?vue-component';
+
+export default {
+  components: {
+    AccountInfo,
+    Plate,
+    Tabs,
+    Auction,
+    History,
+  },
+  props: {
+    name: { type: String, required: true },
+  },
+  data: () => ({ loading: true }),
+  mounted() {
+    const id = setInterval(() => this.updateAuctionEntry(), 3000);
+    this.$once('hook:destroyed', () => clearInterval(id));
+    this.$watch(
+      ({ name }) => name,
+      () => {
+        this.loading = true;
+        this.updateAuctionEntry();
+      },
+      { immediate: true },
+    );
+  },
+  methods: {
+    async updateAuctionEntry() {
+      await this.$watchUntilTruly(() => this.$store.state.middleware);
+      try {
+        const { expiration, bids } = await this.$store.dispatch('names/fetchAuctionEntry', this.name);
+        this.$store.commit('names/setAuctionEntry', {
+          name: this.name,
+          expiration,
+          bids,
+        });
+      } catch (error) {
+        this.$router.push('/names/auctions');
+      }
+      this.loading = false;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../../styles/variables';
+
+.auction {
+  .plate {
+    height: 108px;
+  }
+
+  .tabs {
+    position: sticky;
+    top: calc(146px + env(safe-area-inset-top));
```

Comment:
Should be rebased on top of the develop, there would be new challenges because now you can not do this due to expanding and collapsing ability.

Reply:
Do you mean after #1169 is merged to develop?

Reply:
https://github.com/aeternity/superhero-wallet/pull/1148 is merged now, and we should change the AccountInfo on the top to the AccountSwitcher